### PR TITLE
Add restoreBlockDOM()

### PR DESCRIPTION
**Pull request by taylrj**
Adopt the solution proposed in https://github.com/facebook/draft-js/pull/2228

### CODE REVIEW

File: scripts/module-map.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
 module.exports = Object.assign(
   {
     immutable: 'immutable',
-    React: 'react',
```
Comment (babygoat) [inline scripts/module-map.js:11]: Can you elaborate more on the change?

Comment (taylrj) [reply scripts/module-map.js:11]: Sure, I updated the commit message in https://github.com/twreporter/draft-js/pull/1/commits/862ad3e28b118c38fa46c031669284ad5b01c808.

Please have a see.

Comment (babygoat) [reply scripts/module-map.js:11]: OK, I supposed that that's why the CI of the original repo is broken.

**Comment by babygoat** [comment]
LGTM

**Pull request merged**